IpmiFeaturePkg/Test/FunctionalTest/IpmiShellTest/IpmiShellTest.inf

```diff
@@ -0,0 +1,37 @@
+## @file
+# Functional shell test for IPMI communication.
+#
+# Copyright (c) Microsoft Corporation.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION    = 1.26
+  BASE_NAME      = IpmiShellTest
+  FILE_GUID      = 083541C4-562B-4C5E-AC8E-93A0B8EDA8DE
+  MODULE_TYPE    = UEFI_APPLICATION
+  VERSION_STRING = 1.0
+  ENTRY_POINT    = IpmiTestEntryPoint
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64

```

Comment:
Do we really only support x86 architectures?

Reply:
Nope, just a stale comment. I'm going to just remove this